bin/run-ci

```diff
@@ -23,32 +27,18 @@ sleep 5
 
 $DIR/waitfor-platform
 
-exit 0
-
-### Test Prep
-echo "172.17.0.1 local.astronomer-development.com" | sudo tee -a /etc/hosts
-sudo /tmp/bin/kubectl port-forward -n astronomer svc/astronomer-nginx 80 443 &
-FORWARD_PID=$!
+### Create initial user
+$DIR/create-initial-user "tester@astronomer.io" "password"-
 
-sleep 10
+### Test Platform
+$DIR/test-ap
 
-./bin/create-initial-user "tester@astronomer.io" "password"
-# kill $FORWARD_PID
+exit $?
```

Comment:
Note, I added this here as the tests that follow were not passing, may be more a dns thing than correctness. @jpweber did you say something about the status of these tests?